ajax/libs/onfire.js/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "onfire.js",
+  "filename": "onfire.min.js",
+  "version": "1.0.6",
+  "description": "onfire.js is a simple events dispatcher subscribe / publish library (just 0.9kb). aysnc, simple and usefull.",
```

Comment:
Please remove `(just 0.9kb)` as it may just changed between different versions and we don't frequently update the description.